src/DataLoader/DataLoader.cs

```diff
@@ -85,7 +87,7 @@ private void OnLoadRequested(IDataLoaderTrigger trigger, IDataLoaderContext cont
 			var sequenceId = Interlocked.Increment(ref _sequenceId);
 			PopulateContext(context);
 
-			_ = LoadFromTrigger(cancellationToken, new DataLoaderRequest(sequenceId, trigger, context));
+			_ = LoadFromTrigger(cancellationToken, new DataLoaderRequest(sequenceId, trigger, context, _contextValues));
 
```

Comment:
What's the point of this?

Reply:
We create a local function to be able to use the await keyword because the calling site isn't async.